README.md

```diff
@@ -15,6 +15,9 @@
    - No broken links/buttons.
 4. have consistent uptime.
    - Minor downtime for maintenance or fixing issues is fine
+5. have bot listing requirements (such as, no NSFW outside NSFW channels, respect ratelimites, don't respond to other bots, etc.)
```

Comment:
```suggestion
5. have bot listing requirements (such as no NSFW outside NSFW channels, respect rate-limits, don't respond to other bots, etc.)
```

Reply:
Would we want to do `Have minimum bot requirements (such as but not limited to, no NSFW outside NSFW channels, respect ratelimites, etc.)` or `Have minimum bot requirements.`